.rubocop.yml

```diff
@@ -3,4 +3,7 @@ AllCops:
     - vendor/**/*
     - bin/**/*
 
+Style/DoubleNegation:
```

Comment:
Note that the other double-negation fixes *are* valuable, they just waste CPU cycles, so I am happy you did them!